tethys_gizmos/static/tethys_gizmos/js/select_input.js

```diff
@@ -25,7 +25,11 @@ var TETHYS_SELECT_INPUT = (function() {
  	var initSelectInput;
 
  	initSelectInput = function(tag) {
-        $(tag).select2();
+ 	    $(tag).each(function(){
```

Comment:
Why the $.each here? initSelectInput is already being called once for each instance of the 'select2' class (see line 45ish). Would this be redundant?